src/components/Accordion/Accordion/__tests__/accordion.interactions.js

```diff
@@ -14,41 +14,89 @@ function getOpenedAccordionItem(canvas) {
   return { elPanel, elHeading };
 }
 
-const openCloseAccordionSingleActiveTests = async canvas => {
-  let elHeading, elPanel, before, after;
-  await delay(CHANGES_DELAY);
+function getOpenedAccordionItems(canvas) {
+  const elPanels = canvas.queryAllByRole("region");
+  const elHeadings = elPanels.map(elPanel => {
+    return within(elPanel.parentNode).getByRole("button");
+  });
+  return { elPanels, elHeadings };
+}
 
+async function openAndCheckAccordionItem(canvas, title) {
+  const before = getOpenedAccordionItem(canvas);
+  const elHeading = getAccordionHeadingByText(canvas, title);
+  userEvent.click(elHeading);
+  const elPanel = canvas.getByRole("region");
+  await expect(elHeading.getAttribute("aria-expanded")).toBe("true");
+  await expect(before.elHeading.getAttribute("aria-expanded")).toBe("false");
+  await expect(elHeading.getAttribute("aria-controls")).toBe(elPanel.id);
+}
+
+async function closeAndCheckMultiAccordionItem(canvas, shouldBeOpened) {
```

Comment:
the name `shouldBeOpened` confused me - I thought it was a boolean... how about `expectedOpenedPanels` or something?

Reply:
Changed 👍